sdks/python/container/boot.go

```diff
@@ -73,13 +74,20 @@ const (
 )
 
 func main() {
+	if err := mainError(); err != nil {
+		log.Print(err)
+		os.Exit(1)
+	}
+}
+
+func mainError() error {
```

Comment:
mainError isn't a good name. Convention would call it "tryMain", but that's also a bad name.
If anything, it should be "launchSDKProcess" or similar which documents it's purpose.

Reply:
Ping.

Reply:
Done.